modules/claims_api/app/sidekiq/claims_api/va_notify_declined_job.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VANotifyDeclinedJob < ClaimsApi::ServiceBase
+    LOG_TAG = 'va_notify_declined_job'
+
+    def perform(encrypted_ptcpnt_id:, encrypted_first_name:, poa_code:)
+      lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+      ptcpnt_id = lockbox.decrypt(encrypted_ptcpnt_id)
+      first_name = lockbox.decrypt(encrypted_first_name)
+      poa = find_poa(poa_code:)
+
+      if poa.blank?
+        raise ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
+          detail: "Could not find Power of Attorney with POA code: #{poa_code}"
+        )
+      end
+
+      res = send_declined_notification(ptcpnt_id:, first_name:, poa:)
+
+      ClaimsApi::VANotifyFollowUpJob.perform_async(res.id) if res.present?
+    rescue => e
+      msg = "VA Notify email notification failed to send with error #{e}"
+      slack_client = SlackNotify::Client.new(webhook_url: Settings.claims_api.slack.webhook_url,
+                                             channel: '#api-benefits-claims-alerts',
+                                             username: 'Failed ClaimsApi::VANotifyDeclinedJob')
+
+      slack_client.notify(msg)
+      ClaimsApi::Logger.log(LOG_TAG, detail: msg)
+
+      raise e
+    end
+
+    private
+
+    def find_poa(poa_code:)
+      ClaimsApi::PowerOfAttorney.find do |poa|
+        poa.form_data.dig('serviceOrganization', 'poaCode') == poa_code ||
+          poa.form_data.dig('representative', 'poaCode') == poa_code
+      end
+    end
+
+    def send_declined_notification(ptcpnt_id:, first_name:, poa:)
+      return send_organization_notification(ptcpnt_id:, first_name:) if poa.form_data['serviceOrganization'].present?
+
+      send_representative_notification(ptcpnt_id:, first_name:, poa:)
+    end
+
+    def send_organization_notification(ptcpnt_id:, first_name:)
+      content = {
+        recipient_identifier: ptcpnt_id,
```

Comment:
Per [comment](https://jira.devops.va.gov/browse/API-34448?focusedId=3393591&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-3393591), participant id can be used to send emails.

If testing locally, change this line to `email_address: 'your@email.com',`.